.travis.yml

```diff
@@ -26,6 +26,10 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
     - php: 5.6
+      env: SYMFONY_VERSION=3.1.*
+    - php: 5.6
+      env: SYMFONY_VERSION=3.2.*
+    - php: 5.6
```

Comment:
It's not 1.2.4. See https://travis-ci.org/liip/LiipThemeBundle/jobs/184169726#L151